bartop-api/test/utils/testObjects/drink/index.js

```diff
@@ -88,6 +88,149 @@ const tommyC = {
   ]
 };
 
+const daiquiri = {
+  name: 'Daiquiri',
+  description: 'Classic Cuban cocktail.',
+  instructions: [
+    'Put all ingredients into shaker tin',
+    'Shake ingredients with ice for 10 seconds',
+    'Fine strain into chilled coupe glass'
+  ],
+  ingredients: [
+    {
+      ingredient: 'White rum',
+      amount: {
+        quantity: 2,
+        unit: 'oz'
+      }
+    },
+    {
+      ingredient: 'Lime juice',
+      amount: {
+        quantity: 0.75,
+        unit: 'oz'
+      }
+    },
+    {
+      ingredient: 'Simple syrup',
+      amount: {
+        quantity: 0.75,
+        unit: 'oz'
+      }
+    }
+  ]
+};
+
+const whiskeySour = {
+  name: 'Whiskey Sour',
+  description:
+    'Classic pre-Prohibition cocktail that gets a bad rep but when made well is excellent.',
+  instructions: [
+    'Put all ingredients into shaker tin without ice',
+    'Shake hard for 2 seconds',
+    'Add ice and shake all ingredients for 10 seconds',
+    'Fine strain into chilled glass, either neat or over fresh ice.'
+  ],
+  ingredients: [
+    {
+      ingredient: 'Whiskey',
+      amount: {
+        quantity: 2,
+        unit: 'oz'
+      }
+    },
+    {
+      ingredient: 'Lemon juice',
+      amount: {
+        quantity: 0.75,
+        unit: 'oz'
+      }
+    },
+    {
+      ingredient: 'Simple syrup',
+      amount: {
+        quantity: 0.75,
+        unit: 'oz'
+      }
+    },
+    {
+      ingredient: 'Egg white',
+      amount: {
+        quantity: 1
+      }
+    },
+    {
+      ingredient: 'Angostura bitters (Optional)',
+      amount: {
+        quantity: 1,
+        unit: 'dash'
+      }
+    },
+    {
+      ingredient: 'Lemon peel (Optional)',
+      amount: {
+        quantity: 1,
+        unit: 'for garnish'
+      }
+    }
+  ]
+};
+
+const marg = {
+  name: 'Margarita',
+  description: 'The name means "Daisy" in Spanish.',
+  instructions: [
+    'Put all ingredients into shaker tin',
+    'Shake ingredients with ice for 10 seconds',
+    'Strain into glass over fresh ice or serve neat'
+  ],
+  ingredients: [
+    {
+      ingredient: 'Tequila',
+      amount: {
+        quantity: 2,
+        unit: 'oz'
+      }
+    },
+    {
+      ingredient: 'Cointreau',
+      amount: {
+        quantity: 1,
+        unit: 'oz'
+      }
+    },
+    {
+      ingredient: 'Lime juice',
+      amount: {
+        quantity: 0.5,
+        unit: 'oz'
+      }
+    },
+    {
+      ingredient: 'Simple syrup',
+      amount: {
+        quantity: 0.5,
+        unit: 'oz'
+      }
+    },
+    {
+      ingredient: 'Salt rim',
+      amount: {
+        quantity: 1,
+        unit: 'for garnish'
+      }
+    },
+    {
+      ingredient: 'Lime wheel (Optional)',
+      amount: {
+        quantity: 1,
+        unit: 'for garnish'
+      }
+    }
+  ]
+};
+
```

Comment:
These are great points, thanks for typing them up. Been a while since I thought about how I want the database so I will re-visit that and see about making a table for ingredients. Certainly will look into making it an object too